src/components/auth-module/register-view/register-message/register-message.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+import { translateOptions } from '../../../../i18n/config';
+import { Button, Form } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
+
+class RegisterMessageView extends Component {
+  /***************************
+   *        CONSTRUCTOR
+   ***************************/
+  constructor(props) {
+    super(props);
+  }
+
+  /***************************
```

Comment:
It would be better if we add comments blocks like below whenver require.
```
/***************************
   *         LIFECYCLE
   ***************************/
```